apps/sms/test/marionette/drafts_test.js

```diff
@@ -283,6 +289,34 @@ marionette('Messages Drafts', function() {
       });
     });
 
+    suite('Thread-less recipient-less drafts', function() {
```

Comment:
we changed our terminology from "thread" to "conversation" as this is more understandable and less overloaded...

In Inbox "conversation-less draft" is an item of the list, don't know if we should call this a "conversation" but at least it has the same info :) maybe in Inbox we should use the more general term "inbox item" :)